### PR TITLE
Changes to support MAS 9.0.3..

### PR DESCRIPTION
**Pull request by NatarajBTI**
Changes to support MAS 9.0.3..

### CODE REVIEW

**Comment by NatarajBTI** [comment]
@ocofaigh - Please review this PR. The pipeline run is successful.
This PR has updates for deploying MAS 9.0.3.
Also, a minor correction in displayed variable description is included. So "ibmc-vpc-block-retain-10iops-tier" is changed to "ibmc-vpc-file-500-iops" in description.
I was seeing below error during uninstall & to resolve the same, I have included the runAsUser: 1000 in uninstall Helm chart.

Error: container has runAsNonRoot and image has non-numeric user (root), cannot verify user is non-root (pod: "mas-undeploy-job-qwmzf_mas-uninstall-pipelines(b605c56f-8048-407a-ac16-6f22a7ebc77c)", container: mas-undeploy-job)

File: solutions/existing-cluster/README.md [hunk 1]
```diff
@@ -1,58 +1,25 @@
...
-| <a name="input_storage_class_rwx"></a> [storage\_class\_rwx](#input\_storage\_class\_rwx) | Enter the storage class (read-write many). Enter file storage class for DB2. Default value is ibmc-vpc-block-retain-10iops-tier. Make sure this storage class is present under Storage > StorageClasses section on your Red Hat OpenShift cluster section. | `string` | `"ibmc-vpc-block-retain-10iops-tier"` | no |
-
-## Outputs
-
-| Name | Description |
-|------|-------------|
-| <a name="output_maximo_admin_url"></a> [maximo\_admin\_url](#output\_maximo\_admin\_url) | Admin URL of MAS/Manage application is |
-<!-- END_TF_DOCS -->
+Maximo Application Suite on Existing Red Hat OpenShift Cluster module
+
```
Comment (ocofaigh) [inline solutions/existing-cluster/README.md:2]: What happened to this markdown? This markdown should be automatically generated by the pre-commit hook. You just need to include the following metadata tags, and run `pre-commit run --all-files`

```
<!-- The following content is automatically populated by the pre-commit hook -->
<!-- BEGINNING OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
```

**Review by ocofaigh — CHANGES_REQUESTED** [review]
see above comment

File: solutions/existing-cluster/README.md (hunk 1 above)
Comment (NatarajBTI) [reply solutions/existing-cluster/README.md:2]: yes, I have run the "pre-commit run --all-files" itself..

Comment (ocofaigh) [reply solutions/existing-cluster/README.md:2]: @NatarajBTI it will only work when you have the metadata tags I left in previous comment

Comment (NatarajBTI) [reply solutions/existing-cluster/README.md:52]: @ocofaigh - I have included the metadata tag & have run pre-commit run --all-files & have checked in the changed README.md. Please check the same..thanks

File: solutions/existing-cluster/README.md [hunk 2]
```diff
@@ -48,11 +40,36 @@
...
+### Outputs
 
 | Name | Description |
 |------|-------------|
 | <a name="output_maximo_admin_url"></a> [maximo\_admin\_url](#output\_maximo\_admin\_url) | Admin URL of MAS/Manage application is |
-<!-- END_TF_DOCS -->
+<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->
+Maximo Application Suite on Existing Red Hat OpenShift Cluster module
+
+Requirements
```
Comment (ocofaigh) [inline solutions/existing-cluster/README.md:53]: you need to remove the old code here

Comment (ocofaigh) [reply solutions/existing-cluster/README.md:53]: everything under the `<!-- END OF PRE-COMMIT-TERRAFORM DOCS HOOK -->` tag

File: solutions/existing-cluster/README.md (hunk 1 above)
Comment (NatarajBTI) [reply solutions/existing-cluster/README.md]: @ocofaigh - Issue in README.md is addressed & pipeline tests are successfuly. Please review & merge the same..thanks

**Review by ocofaigh — APPROVED** [review]

**Pull request merged**